CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ x.x.x Release notes (yyyy-MM-dd)
   cannot exist independently from a `Realm`.
 * Aggregate operations are now available on `List`: `min`, `max`, `sum`,
   `average`.
+* Committing write transactions (via `commitWrite` / `commitWriteTransaction` and 
+  `write` / `transactionWithBlock`) now optionally allow for handling errors when
```

Comment:
I'd also put this in the breaking changes section because users will have to update their code for things to work in Swift 2.